### PR TITLE
Use oxrdf & oxrdfio instead of oxigraph

### PR DESCRIPTION
**Pull request by hoijui**
-> Significantly smaller dependencies and shorter compile-time

### CODE REVIEW

File: tests/w3c_testsuite.rs [hunk 1]
```diff
@@ -39,15 +41,15 @@ fn get_remote_file(url: &str) -> Result<String> {
...
 }
 
-fn parse_turtle(url: &str, data: &str) -> Result<Graph> {
-    GraphParser::from_format(GraphFormat::Turtle)
+fn parse_turtle(url: &str, data: &str) -> Result<Dataset> {
+    RdfParser::from_format(RdfFormat::Turtle)
         .with_base_iri(url)?
-        .read_triples(data.as_bytes())?
-        .collect::<std::result::Result<_, _>>()
+        .parse_read(data.as_bytes())
```
Comment (Tpt) [inline tests/w3c_testsuite.rs:46]: To get back a graph:
```suggestion
        .parse_read(data.as_bytes())
        .map(|r| Ok(r?.into()))
```

**Review by Tpt — APPROVED** [review]
Thank you! If you want to simplify the code slightly, there is a way for `parse_turtle` to easily return a `Graph`(see inline comment)

An other option might be to use the `oxttl::TurtleParser` crate directly

File: tests/w3c_testsuite.rs (hunk 1 above)
Comment (hoijui) [reply tests/w3c_testsuite.rs:46]: Hmm.. exactly like this it seems not to work ...
What supports conversion into Graph, exactly?

**Comment by hoijui** [comment]
I now switched oxrdfio -> oxttl, and am not using Dataset anymore

**Comment by Tpt** [comment]
Thank you!

**Pull request merged**